packages/styles/scss/patterns/blocks/leadspace-block/_leadspace-block.scss

```diff
@@ -45,7 +45,7 @@
         padding-bottom: $layout-06;
       }
       @include carbon--breakpoint('lg') {
-        padding-bottom: $layout-07;
+        padding-bottom: $layout-05;
```

Comment:
Thanks for the changes, @andysherman2121! But looks like something is not right. In the `md` breakpoint, following the [visual specs](https://ibm.ent.box.com/file/662154081174), we should use `96px`, as it shows in the document: 
![Captura de Tela 2020-07-02 às 15 03 11](https://user-images.githubusercontent.com/42848561/86395372-87e02d00-bc76-11ea-8b87-23a2c9e730ae.png)

In the breakpoints above (`lg`, `xlg` and `max`), it is `64px`, but visually this value is much bigger than the `96px` from above. Take a look at the proportion: 
![Captura de Tela 2020-07-02 às 15 04 41](https://user-images.githubusercontent.com/42848561/86395611-e9a09700-bc76-11ea-813a-c186d5ca1543.png)

This red rule is showing the difference. I would say that visually the token replaced is much more accurate than the current. Should we ask for help from any of the designers?